src/analysis/classic/druid/feral/modules/features/Abilities.ts

```diff
@@ -0,0 +1,117 @@
+import SPELLS from 'common/SPELLS/classic';
+import CoreAbilities from 'parser/core/modules/Abilities';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    return [
+      // Rotational
+
+      {
+        spell: [SPELLS.MANGLE_CAT.id, ...SPELLS.MANGLE_CAT.lowRanks],
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1000 },
+      },
+      {
+        spell: [SPELLS.SHRED.id, ...SPELLS.SHRED.lowRanks],
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1000 },
+      },
+      {
+        spell: [SPELLS.RAKE.id, ...SPELLS.RAKE.lowRanks],
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1000 },
+      },
+      {
+        spell: [SPELLS.SAVAGE_ROAR.id],
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1000 },
```

Comment:
does feral in Wrath have any attack speed / haste (unsure of proper term in classic) scaling of their gcds in cat form? I know in retail none of the energy specs have scaling gcds (which would be `gcd: { static: 1000 }` instead of `gcd: { base: 1000 }`).

Reply:
I believe it works the same, I'm pretty sure haste only effects our energy regen - I swap them to static

Reply:
haste doesn't affect gcd, nor the energy regen (which is fixed to 10/sec)
only thing haste does for feral is increase melee speed.

Reply:
Thanks for the clarification :)